lib/utils/execution-state-manager.js

```diff
@@ -18,10 +18,10 @@ class ExecutionStateManager {
     this._completedBrowsers = 0;
 
     // An array of modules to be load balanced to a number of browsers
-    this._sharedModuleQueue = null;
+    this._testModuleQueue = null;
 
     // A map of browserId to an array of modules to be passed to the browser
```

Comment:
can we add to the description that this should only be populated when `--replay-execution` is passed?
```
// A map of browserId to an array of test modules. This should only be populated by `--replay-execution`
```